python/ql/src/semmle/python/dataflow/new/internal/DataFlowPublic.qll

```diff
@@ -60,7 +60,16 @@ newtype TNode =
    */
   TKwUnpacked(CallNode call, CallableValue callable, string name) {
     call_unpacks(call, _, callable, name, _)
-  }
+  } or
+  /**
+   * A synthetic node representing that an iterable sequence flows to consumer.
+   */
+  TIterableSequence(UnpackingAssignmentTarget consumer) { consumer instanceof SequenceNode } or
+  /**
+   * A synthetic node representing that there may be an iterable element
+   * for `consumer` to consume.
+   */
+  TIterableElement(UnpackingAssignmentTarget consumer)
```

Comment:
For most of these disjuncts, we have names of the form `...Node` (though notably `TKwUnpacked` slipped through the cracks somehow). Should we perhaps endeavour to keep this consistent?

Reply:
Sure, that does seem to be a convention :-)